mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -44,7 +44,7 @@ def process_model_config(model_cfg: mmcv.Config,
             trans_type = trans['type']
             if trans_type == 'Resize':
                 trans['keep_ratio'] = False
-            elif trans_type == 'Pad':
+            elif trans_type == 'Pad' and 'size_divisor' in trans:
```

Comment:
Is there any case that  `trans_type == 'Pad' and 'size_divisor' not in trans`?

Reply:
https://github.com/open-mmlab/mmdetection/blob/master/configs/efficientnet/retinanet_effb3_fpn_crop896_8x4_1x_coco.py

Reply:
In case "type='Pad', size=(896, 896)", I think we should keep `keep_ratio` of `Resize` as the original one, shouldn't we? especially considering dump transforms to SDK metafile